packages/components/src/draggable/index.native.js

```diff
@@ -77,25 +77,39 @@ const Draggable = ( { children, onDragEnd, onDragOver, onDragStart } ) => {
 		}
 	);
 
+	function getFirstTouchEvent( event ) {
+		'worklet';
+
+		const sortedEventsById = event.allTouches.sort(
+			( a, b ) => b.id + a.id
+		);
+		return sortedEventsById[ 0 ];
+	}
+
 	const panGesture = Gesture.Pan()
 		.manualActivation( true )
 		.onTouchesDown( ( event ) => {
-			const { x = 0, y = 0 } = event.allTouches[ 0 ];
+			const firstEvent = getFirstTouchEvent( event );
+
+			const { x = 0, y = 0 } = firstEvent;
 			initialPosition.x.value = x;
 			initialPosition.y.value = y;
 		} )
-		.onTouchesMove( ( _, state ) => {
+		.onTouchesMove( ( event, state ) => {
 			if ( ! isPanActive.value && isDragging.value ) {
 				isPanActive.value = true;
 				state.activate();
 			}
-		} )
-		.onUpdate( ( event ) => {
-			lastPosition.x.value = event.x;
-			lastPosition.y.value = event.y;
 
-			if ( onDragOver ) {
-				onDragOver( event );
+			if ( isPanActive.value && isDragging.value ) {
+				const firstEvent = getFirstTouchEvent( event );
```

Comment:
I'm wondering if we could keep the id of the first touch and use it here instead of calculating it on every update, wdyt?

Reply:
That's a great suggestion! 😎  I'll update the PR with the change!

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/40519/commits/d1046c901b01500cadf28beb26c9012800daf3a7